app/controllers/posts_controller.rb

```diff
@@ -23,7 +23,8 @@ def edit
   def create
     @post = Post.new(post_params)
     @post.type_id = 1
-
+    @post.user_id = current_user.id
+    puts @post.user_id
```

Comment:
Do you need a `puts` here?

Reply:
I forgot that, thanks I´ve alredy delete it.